content/kart og reiser.md

```diff
@@ -1,8 +1,8 @@
 +++
-title = "Petter"
+title = "Kart og reiser"
 lastmod = 2020-08-12T12:00:00
 [menu]
-main = { weight = 10 }
+main = { weight = 40 }
```

Comment:
@Stigjb Har du sett at jeg bruker disse "tomme" sidene for å skape en naturlig rekkefølge i menyen? Har du noen kommentarer?